NuRadioReco/modules/voltageToEfieldConverter.py

```diff
@@ -136,6 +136,10 @@ def run(self, evt, station, det, debug=False, debug_plotpath=None, use_channels=
             be save into the `debug_plotpath` directory
         use_channels: array of ints
             the channel ids to use for the electric field reconstruction
+        use_MC_direction:
```

Comment:
I know you didn't add the parameter, but can you document it as well, please.

Reply:
I have added this documentation.